spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpNetConnection.java

```diff
@@ -103,10 +104,11 @@ public synchronized void send(Message<?> message) throws Exception {
 			this.socketOutputStream = new BufferedOutputStream(this.socket.getOutputStream(),
 					writeBufferSize > 0 ? writeBufferSize : 8192);
 		}
-		Object object = this.getMapper().fromMessage(message);
+		Object object = getMapper().fromMessage(message);
+		Assert.state(object != null, "Mapper mapped the message to 'null.");
```

Comment:
Missed closing quote:
```suggestion
		Assert.state(object != null, "Mapper mapped the message to 'null'.");
```